scss/mixins/_grid-framework.scss

```diff
@@ -4,47 +4,40 @@
 // any value of `$grid-columns`.
 
 @mixin make-grid-columns($columns: $grid-columns, $gutter: $grid-gutter-width, $breakpoints: $grid-breakpoints) {
-  // Common properties for all breakpoints
-  %grid-column {
-    position: relative;
-    // Prevent columns from collapsing when empty
-    min-height: 1px;
-    // Inner gutter via padding
-    padding-left: ($gutter / 2);
-    padding-right: ($gutter / 2);
-  }
   $breakpoint-counter: 0;
   @each $breakpoint in map-keys($breakpoints) {
     $breakpoint-counter: ($breakpoint-counter + 1);
-    @for $i from 1 through $columns {
-      .col-#{$breakpoint}-#{$i} {
-        @extend %grid-column;
-      }
-    }
     @include media-breakpoint-up($breakpoint, $breakpoints) {
-      // Work around cross-media @extend (https://github.com/sass/sass/issues/1050)
-      %grid-column-float-#{$breakpoint} {
-        float: left;
+      @if $enable-flex {
+        .col-#{$breakpoint} {
+          position: relative;
+          max-width: 100%;
+          min-height: 1px;
+          flex-basis: 0;
```

Comment:
Properties should be ordered position, max-width, min-height, padding-right, padding-left, flex-basis, flex-grow